apps/analytics/src/modules/__init__.py

```
@@ -0,0 +1,8 @@
+from .participant_analytics import *
```

Ruff (0.8.2)

`from .participant_analytics import *` used; unable to detect undefined names

(F403)